src/dials/algorithms/integration/boost_python/integration_integrator_ext.cc

```diff
@@ -125,13 +128,11 @@ namespace dials { namespace algorithms { namespace boost_python {
     data.resize(bbox_new.size());
 
     // Reorder the reflections
-    af::boost_python::flex_table_suite::reorder(data, indices.const_ref());
+    reorder(data, indices.const_ref());
```

Comment:
Although... since it is used exactly once (or twice, for below), it's probably clearer not to use `using` but to qualify it at the point of call.